_conformance/conformance_proto/conformance.proto

```diff
@@ -1,285 +0,0 @@
-// Protocol Buffers - Google's data interchange format
```

Comment:
IIUC, the removal of this file means golden_test.go won't catch when the generated .pb.go drifts from one generated from the version in the proto repo. It seems to me that we should either:
1) avoid checking in `conformance.pb.go` and rely on the make script to automatically regenerate it upon execution
2) checking in both the `.proto` and `.pb.go` and add a test to ensure that the checked-in and upstream `.proto` files remain the same.
3) add a test to ensure the checked in `.pb.go` doesn't drift when dynamically regenerated from upstream.

Reply:
Checked in the .proto, made "make test" in the conformance directory warn if its out of date.